src/infra/repository/firestore/members.rs

```diff
@@ -2,10 +2,11 @@ use std::sync::Arc;
 
 use async_trait::async_trait;
 use firestore::{paths, struct_path, FirestoreDb};
+use serenity::futures::StreamExt;
```

Comment:
TODO: fix this import

Reply:
https://github.com/approvers/members-db/pull/2/commits/478ceca592d9d0dc26ba446b8cc00efc7e879c16